bundles/org.platformio.eclipse.ide.home.ui/META-INF/MANIFEST.MF

```diff
@@ -15,7 +15,10 @@ Require-Bundle: org.eclipse.core.runtime;bundle-version="0.0.0",
  org.eclipse.ui.editors;bundle-version="0.0.0",
  org.eclipse.core.resources;bundle-version="0.0.0",
  org.platformio.eclipse.ide.home;bundle-version="0.0.0",
- org.eclipse.ui.browser;bundle-version="0.0.0"
+ org.eclipse.ui.browser;bundle-version="0.0.0",
+ org.eclipse.ui.console;bundle-version="0.0.0",
+ org.eclipse.text;bundle-version="0.0.0",
+ org.eclipse.jface.text;bundle-version="3.17.0"
```

Comment:
we need to keep one style for dependencies
1) let's use  `bundle-version="0.0.0"` if we don't have specific requirements
2) let's support alphabetical order for dependencies